samples/bluetooth/iso_broadcast_benchmark/src/broadcaster.c

```diff
@@ -613,13 +618,59 @@ static void iso_timer_timeout(struct k_work *work)
 	seq_num++;
 }
 
+static uint16_t calc_adv_interval(uint32_t sdu_interval_us)
+{
+	/* Default to 6 times the SDU interval and then reduce until we reach a reasonable maximum
+	 * advertising interval (BT_GAP_PER_ADV_SLOW_INT_MAX)
+	 * sdu_interval_us can never be larger than 1048ms (BT_ISO_SDU_INTERVAL_MAX)
+	 * so a multiple of it will always be possible to keep under BT_GAP_PER_ADV_SLOW_INT_MAX
+	 * (1200ms)
+	 */
+
+	/* Convert from microseconds to advertising interval units (0.625ms)*/
+	const uint32_t interval = BT_GAP_US_TO_ADV_INTERVAL(sdu_interval_us);
+	uint32_t adv_interval = interval * 6U;
+
+	/* Ensure that the adv interval is between BT_GAP_PER_ADV_FAST_INT_MIN_1 and
+	 * BT_GAP_PER_ADV_SLOW_INT_MAX for the sake of interopability
+	 */
+	while (adv_interval < BT_GAP_PER_ADV_FAST_INT_MIN_1) {
+		adv_interval += interval;
+	}
+
+	while (adv_interval > BT_GAP_PER_ADV_SLOW_INT_MAX) {
+		adv_interval -= interval;
+	}
+
+	/* If we cannot convert back then it's not a lossless conversion */
+	if (BT_GAP_ADV_INTERVAL_TO_US(interval) != sdu_interval_us) {
+		LOG_WRN("Advertising interval of 0x04%x is not a multiple of the advertising "
```

Comment:
Can we change this warning to `LOG_INF`? Other controllers may not have this limitation.

Reply:
Because of the randomness of the advertising event, a controller may "move" the advertising events in such a way that they do not overlap with other ISO-activities. This is possible because the randomness is in the range of 10 ms

Reply:
@cvinayak is this something we could do with ZLL? 

> This is possible because the randomness is in the range of 10 ms

But if it is random, how can the controller move things? Wouldn't that make it non-random?

Reply:
"Random enough" is the key here. It is possible to move the advertising event slightly and still preserve some randomness

Reply:
BIG event lengths are larger than 10 ms in high reliability configurations, having an optimal advertising interval (and separate primary and aux channel event scheduling) aids a deterministic and quick ISO synchronization under such cases.

>is this something we could do with ZLL?

@Thalley This logging is redundant as https://github.com/zephyrproject-rtos/zephyr/pull/80733 ensures BIG events are not skipped.

Reply:
> @Thalley This logging is redundant as #80733 ensures BIG events are not skipped.

So we do not "need" to have the ISO interval as a multiple of the ADV interval?

Reply:
Correction, I got confused, this is related to SDU interval not in multiple of 0.625 ms. If unframed ISO SDU in use, yes the log is a good suggestion to the user which will ensure all periodic advertising PDU do not overlap and hence faster periodic sync established.

Reply:
I think it makes sense to not log this for framed. Will add a check for framing. 

For framed, can/should the host do anything?

Reply:
When framed is used, the controller should be able to select a "nice" ISO interval similar to https://github.com/zephyrproject-rtos/zephyr/pull/81227#discussion_r1847867086